libs/core/src/main/java/org/opensearch/core/common/util/CollectionUtils.java

```diff
@@ -92,7 +102,14 @@ public static <T> List<T> rotate(final List<T> list, int distance) {
     }
 
     /**
-     * in place de-duplicates items in a list
+     * In place de-duplicates items in a list<p>
```

Comment:
```
> Task :libs:opensearch-core:compileJava
/var/jenkins/workspace/gradle-check/search/libs/core/src/main/java/org/opensearch/core/common/util/CollectionUtils.java:106: warning: empty <p> tag
     * Noop if the list is empty or has one item.<p>
```

Reply:
@ker2x this is still the issue that fails the build

Reply:
this is frustrating because i don't have the problem when i do it in local. i'm running another batch of test and pushing

Reply:
pushed

Reply:
> this is frustrating because i don't have the problem when i do it in local. i'm running another batch of test and pushing

Please use JDK-20 locally

Reply:
It is not fixes:
```
> Task :libs:opensearch-core:compileJava
/var/jenkins/workspace/gradle-check/search/libs/core/src/main/java/org/opensearch/core/common/util/CollectionUtils.java:107: warning: empty <p> tag
     * <p>
       ^
```

Reply:
@reta thank you. jdk updated to 20. i see the warning. i'll be able to test properly now. Will push a fix after running the the tests.